source/reference/minio-mc/mc-support-perf.rst

```diff
@@ -0,0 +1,224 @@
+===================
+``mc support perf``
+===================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support perf
+
+Description
+-----------
+
+Use the :mc-cmd:`mc support perf` command to analyze the performance of the S3 read/write, network I/O, or drive read/write.
```

Comment:
Rethinking this a bit:

```suggestion
Use the :mc-cmd:`mc support perf` command to perform performance tests for S3 API (read/write), network IO, and storage (drive read/write)

```
Thoughts?

Reply:
Really don't like `perform performance`. :|
The rest is good, though.

Going with this for now unless one of us comes up with something better. 
> Use the :mc-cmd:`mc support perf` command to review the performance of the S3 API (read/write), network IO, and storage (drive read/write).